service/init.go

```diff
@@ -375,14 +286,70 @@ func (amf *AMF) Start() {
 		logger.InitLog.Errorf("initialise kafka stream failed, %v ", err.Error())
 	}
 
-	self := context.AMF_Self()
+	self := amfContext.AMF_Self()
 	util.InitAmfContext(self)
 	if self.EnableDbStore {
 		self.Drsm, err = util.InitDrsm()
 		if err != nil {
 			logger.InitLog.Errorf("initialise DRSM failed, %v", err.Error())
 		}
 	}
+	ctx, cancelServices := ctxt.WithCancel(ctxt.Background())
+
+	if factory.AmfConfig.Configuration.Telemetry != nil && factory.AmfConfig.Configuration.Telemetry.Enabled {
+		var tp *sdktrace.TracerProvider
+		tp, err = tracing.InitTracer(ctx, tracing.TelemetryConfig{
+			OTLPEndpoint:   factory.AmfConfig.Configuration.Telemetry.OtlpEndpoint,
+			ServiceName:    "amf",
+			ServiceVersion: factory.AmfConfig.Info.Version,
+			Ratio:          *factory.AmfConfig.Configuration.Telemetry.Ratio,
+		})
+		if err != nil {
+			logger.InitLog.Panic("could not initialize tracer", zap.Error(err))
+		}
+		logger.InitLog.Infoln("tracer initialized successfully")
+		defer func() {
+			err = tp.Shutdown(ctx)
+			if err != nil {
+				logger.InitLog.Error("failed to shutdown tracer", zap.Error(err))
+			} else {
+				logger.InitLog.Infoln("tracer shutdown successfully")
+			}
+		}()
+	}
+
+	registrationChan := make(chan []nfConfigApi.AccessAndMobility, 1)
+	contextUpdateChan := make(chan []nfConfigApi.AccessAndMobility, 1)
```

Comment:
```suggestion
	registrationChan := make(chan []nfConfigApi.AccessAndMobility, 100)
	contextUpdateChan := make(chan []nfConfigApi.AccessAndMobility, 100)
```
We need to increase the channel buffer to a reasonable number as we use `select case` if the buffer is full, message will be lost.  Increasing channel buffer will help here to store the configs which are waiting for processing.